index.js

```diff
@@ -18,6 +18,7 @@ const wrapWithHeaderAndFooter = (outputLines, electronVersion) => {
   const newOutputLines = []
   utils.extendArray(newOutputLines, fs.readFileSync(path.resolve(__dirname, 'base/base_header.ts'), 'utf8').replace('<<VERSION>>', electronVersion).split(/\r?\n/))
 
+  newOutputLines.push('/** @deprecated */')
```

Comment:
Don't think we want to deprecate this, we can quite easily have both 👍

Reply:
If the module itself doesn't declare a global variable, neither should the typings.

Reply:
@NaridaL Technically electron does make magic globals, for instance `process.windowsStore` and other such things.  I'm happy to add this new way of doing things in if that's what people want but not keen on "deprecating" the old way 👍